web/package.json

```diff
@@ -143,7 +143,7 @@
     "autoprefixer": "10.4.12",
     "colors": "^1.4.0",
     "css-mediaquery": "0.1.2",
-    "cypress": "10.10.0",
+    "cypress": "12.7.0",
```

Comment:
Nice! This will close one of our draft issues as well.

Reply:
Do we need to do any more steps for migrating here?
- https://docs.cypress.io/guides/references/migration-guide#Vite-Dev-Server-cypressvite-dev-server

Reply:
Seems to just work as expected. I think we'd need changes if we wanted the tests to run with a custom test specific Vite config